submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/model/DocSubmissionDraft.kt

```diff
@@ -0,0 +1,21 @@
+package ac.uk.ebi.biostd.persistence.doc.model
+
+import org.bson.types.ObjectId
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+
+@Document("submission-drafts")
+data class DocSubmissionDraft(
+    @Id
+    val id: String,
+    val userId: String,
+    val key: String,
+    val content: String
+) {
+    constructor(userId: String, key: String, content: String) : this(ObjectId().toString(), userId, key, content)
+
+    companion object {
+        const val USER_ID = "userId"
```

Comment:
Constants outside the class, please. Unless we're changing the "coding standard" we've been using.